app/assets/stylesheets/application.scss

```diff
@@ -132,6 +132,18 @@ a:hover {
   color: $font-color;
 }
 
+.photo{
+  padding-top: 5px;
+  width: 40px;
+  height:40px;
+  overflow: hidden;
+}
+
+#logo{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space